.github/workflows/build.yml

```diff
@@ -144,3 +144,115 @@ jobs:
             artifact/*.tar.gz
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build-other-mac:
```

Comment:
create a standalone yaml for mac build. artifact downloaded will mix up

Reply:
OK, todo

Reply:
ok, add todo